be/src/olap/task/engine_storage_migration_task.cpp

```diff
@@ -262,7 +262,7 @@ OLAPStatus EngineStorageMigrationTask::_copy_index_and_data_files(
     for (auto& rs : consistent_rowsets) {
         status = rs->copy_files_to(schema_hash_path);
         if (status != OLAP_SUCCESS) {
-            if (remove_all_dir(schema_hash_path) != OLAP_SUCCESS) {
+            if (!FileUtils::remove_all(schema_hash_path).ok()) {
```

Comment:
get the status and log it if it is not OK
we prefer not do operation and check in `if` clause